creed/tiling_utils_test.py

```diff
@@ -22,6 +24,40 @@
 _AUTO_MOLY_INTERVAL_SETTING_CASES = [False, True]
 _AUTO_MOLY_INTERVAL_VALUE_CASES = [3, 4]
 
+# for TMA rhombus support, the order of the tests:
+# 1. rectangular grid
+# 2. slant only along the x-axis
+# 3. slant only along the y-axis
+# 4. slant along the x- and y-axes
+# make the slants along the x- and y-axis different for completeness of test 4
+# TODO: we need to clean up this testing format, refer to Adam's setup
+_TMA_RHOMBUS_TEST_NAMES = {
```

Comment:
Although this might move to a pytest-cases format, a quick utility function to generate most of these would probably be good.  Then you should be able to freely edit those as needed:

```
def _gen_test_corners():
    return [
        tiling_utils.XYCoord(1500, 2000),
        ...
    ]

X_SLANT = _gen_test_corners()
X_SLANT[2].x += 100
X_SLANT[3].x += 200

...


_TMA_RHOMBUS_TEST_NAMES = {
    'rectangle' : _gen_test_corners(),
    'x-slant': X_SLANT,
    ...
}

## or in cases format

class RhombusTestCoords:
    ...

    def case_xslant():
        coords = _gen_test_corners()
        coords[2].x += 100
        coords[3].x += 200
        
        ... 

```